lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -184,16 +184,11 @@ describe('LMSFilePicker', () => {
 
     // Wait for the initial file fetch to fail.
     const wrapper = renderFilePicker();
-    try {
-      await fakeApiCall;
-    } catch {
-      /* unused */
-    }
     wrapper.update();
```

Comment:
This `wrapper.update()` should now be redundant, assuming `renderFilePicker` returns a "current" wrapper at the end, which is what I would expect as a reader.

Reply:
Fixed!